website/docs/guides/best-practices/how-we-structure/2-staging.md

```diff
@@ -0,0 +1,227 @@
+  - ❌ `stg_[entity].sql` - might be specific enough at first, but will break down in time. Adding the source system into the file name aids in discoverability, and allows understanding where a component model came from even if you aren't looking at the file tree.
+  - ✅ **Plural.** SQL, and particularly SQL in dbt, should read as much like prose as we can achieve. We want to lean into the broad clarity and declarative nature of SQL when possible. As such, unless there’s a single order in your `orders` table, plural is the correct way to describe what is in a table with multiple rows.
+
+### Staging: Models
+
+Now that we’ve got a feel for how the files and folders fit together, let’s look inside one of these files and dig into what makes for a well structured staging model.
+
+Below, is an example of a standard staging model (from our `stg_stripe__payments` model) that illustrates the common patterns within the staging layer. We’ve organized our model into two [CTEs](https://docs.getdbt.com/terms/cte): one pulling in a source table via the [source macro](https://docs.getdbt.com/docs/building-a-dbt-project/using-sources#selecting-from-a-source) and the other applying our transformations.
+
+While our later layers of transformation will vary greatly from model to model, every one of our staging models will follow this exact same pattern. As such, we need to make sure the pattern we’ve established is rock solid and consistent.
+
+```sql
+-- stg_stripe__payments.sql
+
+with
+
+source as (
+
+    select * from {{ source('stripe','payment') }}
+
+),
+
+renamed as (
+
+    select
+        -- ids
+        id as payment_id,
+        orderid as order_id,
+
+        -- strings
+        paymentmethod as payment_method,
+        case
+            when payment_method in ('stripe', 'paypal', 'credit_card', 'gift_card') then 'credit'
+            else 'cash'
+        end as payment_type,
+        status,
+
+        -- numerics
+    amount as amount_cents,
+        amount / 100.0 as amount,
+        
+        -- booleans
+        case
+            when status = 'successful' then true
+            else false
+        end as is_completed_payment,
+
+    -- dates
+    date_trunc('day', created) as created_date,
+
+        -- timestamps
+        created::timestamp_ltz as created_at
+
+    from source
+
+)
+
+select * from renamed
+```
+
+- Based on the above, the most standard types of staging model transformations are:
+  - ✅ **renaming**
```

Comment:
Potentially capitalize the first letter of the bullets to be consistent with other bullets/enhance readability